src/modules/tasks.js

```diff
@@ -1,14 +1,50 @@
-const tasks = [
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'complete To Do list project',
-    completed: false,
-    index: 1,
-  },
-];
-
-export default tasks;
+import { getTasks, saveTasks } from './localStorage.js';
+
+const tasksContainer = document.querySelector('.tasks');
+const taskTemplate = document.getElementById('task-template');
+
+export default class Tasks {
+  constructor() {
+    this.getData();
+  }
+
+  getData() {
+    this.taskArray = getTasks();
+  }
+
+  addTask(task) {
+    this.taskArray.push(task);
+    saveTasks(this.taskArray);
+  }
+
+  removeTask(index) {
+    this.taskArray.splice(index, 1);
+    for (let i = 0; i < this.taskArray.length; i += 1) {
+      this.taskArray[i].index = i + 1;
+    }
+    saveTasks(this.taskArray);
+  }
+
```

Comment:
- Hey, I can't seem to delete a task because there seems not to be a delete button from your app. please, kindly fix this.